pandas/core/frame.py

```diff
@@ -7623,13 +7618,24 @@ def aggregate(self, func=None, axis: Axis = 0, *args, **kwargs):
         return result
 
     def _aggregate(self, arg, axis: Axis = 0, *args, **kwargs):
+        from pandas.core.apply import frame_apply
+
+        op = frame_apply(
```

Comment:
let's consider having this *inside* the `frame_apply` in the future and just passing the obj (e.g. the transpose / reverse is internal to `frame_apply`)